src/Cosmo.php

```diff
@@ -256,18 +256,29 @@ public function quote(string $quote): string
     /**
      * @param float $value
      * @param string $currency The 3-letter ISO 4217 currency code indicating the currency to use.
+     * @param int|null $precision The needed number of decimals digits 
      * @param string|null $pattern
      * @return string
      * @throws Exception
      */
-    public function money(float $value, string $currency = null, string $pattern = ''): string
+    public function money(float $value, string $currency = null, int $precision= -1,  string $pattern = ''): string
     {
         $currency = $currency ?: $this->modifiers['currency'];
         if (!$currency) {
             throw new Exception("No currency is set to format the monetary value. Set the region subtag in the local identifier (e.g. en -> en_AU) 
                                                     or provide a valid currency code parameter.");
         }
-        return (new NumberFormatter($this->locale, NumberFormatter::CURRENCY, $pattern))->formatCurrency($value, $currency);
+
+        if (!is_int($precision)) {
```

Comment:
The $currency variable is typed as in in the function signature. Do we still need this check?

Reply:
You are right. old habits die hard :)